flutter_local_notifications/android/src/main/java/com/dexterous/flutterlocalnotifications/FlutterLocalNotificationsPlugin.java

```diff
@@ -101,6 +105,8 @@
     private static final String INVALID_BIG_PICTURE_ERROR_CODE = "INVALID_BIG_PICTURE";
     private static final String INVALID_SOUND_ERROR_CODE = "INVALID_SOUND";
     private static final String INVALID_LED_DETAILS_ERROR_CODE = "INVALID_LED_DETAILS";
+    private static final String GET_ACTIVE_NOTIFICATIONS_ERROR_CODE = "GET_ACTIVE_NOTIFICATIONS_ERROR_CODE";
+    private static final String GET_ACTIVE_NOTIFICATIONS_ERROR_MESSAGE = "Android version must be at least 23 to use getActiveNotifications";
```

Comment:
23 is the API level, 6.0 is the Android version. Could you update this error to refer to version number as that's what I've been using in the plugin as there will be developers who haven't worked on Android directly to be familiar with API levels

Reply:
Sure, maybe we can discuss the exact wording here before I push changes?

I'm not sure I understood which number you prefer: the API level, or the Android version? I've searched for "Android" in your repository and the only occurence I could find was the README that says 
```markdown
**Android API 16+** (4.1+, the minimum version supported by Flutter).
```
so should I replace it with (version 1):
```java
"Android API Level must be at least 23 to use getActiveNotifications"
```
or with (version 2)
```java
"Android version must be at least 6.0 to use getActiveNotifications"
```

Reply:
Sorry for the confusing. You're right that the readme has both in there though for public API docs I've simplified it refer to Android version i.e. go with version 2